lib/cucumber/rake/task.rb

```diff
@@ -5,6 +5,7 @@
   # Support Rake > 0.8.7
   require 'rake/dsl_definition'
 rescue LoadError
+  warn 'LoadError for rake/dsl_definition'
```

Comment:
This is one of those place where we _want_ to silence the LoadError and not caring about it.

For ancient Rake versions before that version, this file won't exist (and won't be necessary) and Ruby will raise the LoadError. We catch that and silence that error, telling Ruby: "This is normal."

So, we want to silence RuboCop's complaining about this specific LoadError silencing.

Reply:
I've pushed a new version without this. :)